VimeoNetworking/VimeoNetworking/Request.swift

```diff
@@ -210,10 +151,13 @@ public struct Request<ModelType: MappableResponse>
     {
         var URI = self.path
         
-        let queryString = AFQueryStringFromParameters(self.parameters)
-        if queryString.characters.count > 0
+        if let parameters = self.parameters as? VimeoClient.RequestDictionary
         {
-            URI += "?" + queryString
+            let queryString = AFQueryStringFromParameters(parameters)
+            if queryString.characters.count > 0
+            {
+                URI += "?" + queryString
```

Comment:
Probably. This was exiting copy and pasted logic though so I'm going to keep it as is for now